deploy/ocp-templates/glusters3template.json

```diff
@@ -0,0 +1,205 @@
+{
+    "kind": "Template",
+    "apiVersion": "v1",
+    "metadata": {
+        "name": "glusters3template",
+        "creationTimestamp": null,
+        "annotations": {
+          "description": "Gluster s3 service template",
+          "tags": "glusterfs,heketi"
```

Comment:
Actually we depend upon heketi to create volume..although internally. will add gluster-s3 tag as well.